deploy.sh

```diff
@@ -44,7 +44,11 @@ cd $DEST_DIR
 # Transform the readme
 if [ -f readme.md ]; then
 	mv readme.md readme.txt
-	sed -i '' -e 's/^# \(.*\)$/=== \1 ===/' -e 's/ #* ===$/ ===/' -e 's/^## \(.*\)$/== \1 ==/' -e 's/ #* ==$/ ==/' -e 's/^### \(.*\)$/= \1 =/' -e 's/ #* =$/ =/' readme.txt
+	if [ -f CHANGELOG.md ]; then
```

Comment:
An interesting idea (having it be case-insensitive) however I think the issue is that the file system itself is case sensitive.

The best way to make it support both lowercase and uppercase filenames would be to code for both. Either by duplicating the code for both instances; or by turning the file paths into variables set based on which is found?